src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AltSvc.cs

```diff
@@ -81,11 +81,6 @@ fromVersion.Major switch
         [ConditionalFact]
         public async Task AltSvc_ConnectionFrame_UpgradeFrom20_Success()
         {
-            if (UseVersion == HttpVersion30)
```

Comment:
Is this intentional? Is the test fixed?

Reply:
Ah nice catch, thanks! Nope, this is not intentional. Will fix it